docs/src/components/DocBlock.astro

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
New component for rendering blocks of documentation such as files, mixins, functions, etc.